account_banking_fr_lcr/models/account_payment_order.py

```diff
@@ -31,7 +31,9 @@ def _prepare_field(self, field_name, field_value, size):
                 ]
             for unallowed_ascii_char in unallowed_ascii_chars:
                 value = value.replace(unallowed_ascii_char, '-')
-        except:
+        except BaseException:
```

Comment:
In my understanding, in most case it should be `Exception` instead of `BaseException`, because `BaseException` also includes system exiting exceptions.

Reply:
@sbidoul that's what I thought too but @yvaucher showed me that `python -m autopep8 --select=E722 -a test.py` replace `except:` by `except BaseException:`

Reply:
I read something wrong without checking :) `except Exception` does not raise E722 hopefully, I fixed that.

Reply:
It appears we cheched with @guewen that E722 isn't raised with `except Exception:` so it the solution to go.

I think that autopep8 does it to make it exactly the same as `except:` but it is a bad idea in most case to keep it that way for the reason @sbidoul explained.